manifests/instance.pp

```diff
@@ -73,6 +78,7 @@
   $status             = $elasticsearch::status,
   $config             = undef,
   $configdir          = undef,
+  $manage_datadir     = true,
```

Comment:
When using $elasticsearch::manage_datadir as a default value for $manage_datadir in the instance define it allows you to either set it to false for every instance you create, or you can override it per individual instance.
This way if the user has 2 or more instances, the value doesn't have to be duplicated for each instance.